### ISSUE TITLE
Only first occurrence of template variable gets replaced

### ISSUE THREAD

**Issue by Pikktee** [phibr0/obsidian-dictionary#13]
Playing around with the template functionality I noticed that only the first occurrence of a placeholder gets replaced.

Given the following template...
```
## {{word}} (Glossary)
?
{{meanings}}

—
**Source:**
https://www.google.de/search?q=Wörterbuch#dobs={{word}}
```

...second `{{word}}` placeholder doesn't get replaced currently.

**Comment by phibr0** [comment]
Just fixed this, if you update to v2.9.8 it will replace all your variables!

**Comment by Pikktee** [comment]
Thank you very much! Working great now...  👍
